account_invoice_supplier_duplicate_ref/tests/test_account_invoice_supplier_duplicate_ref.py

```diff
@@ -0,0 +1,4 @@
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+
+# TODO:
```

Comment:
?

Reply:
I add this section [https://github.com/OCA/account-invoicing/pull/535/files#diff-a2bb309a44ef9db6153ec476db4d06cfR2](https://github.com/OCA/account-invoicing/pull/535/files#diff-a2bb309a44ef9db6153ec476db4d06cfR2)